config/config.sample.yaml

```diff
@@ -22,35 +45,25 @@ rtm:
   enable: true
   logging: "silent" # Logging level specific to RTM traffic.
 
+## Start: Optional options if using RTM API, required otherwise.
```

Comment:
This section boundary is a weird notation and not used anywhere else in this file. As it only applies to two options, can we just put the optional explanation on each?